tests/helpers.py

```diff
@@ -1138,6 +1138,93 @@ def get_master_nodes():
     return master_nodes_list
 
 
+def get_snapshot_time(snap_name, status='start'):
+    """
+    Get the starting/ending creation time of a PVC based on provisioner logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pvc_name (str / list): Name of the PVC(s) for creation time
+                               the list will be list of pvc objects
+        status (str): the status that we want to get - Start / End
+
+    Returns:
+        datetime object: Time of PVC(s) creation
+
+    """
+
+    def get_pattern_time(log, snapname, pattern):
+        """
+        Get the time of pattern in the log
+
+        Args:
+            log (list): list of all lines in the log file
+            snapname (str): the name of hte snapshot
+            pattern (str): the pattern that need to be found in the log (start / bound)
+
+        Returns:
+            str: string of the pattern timestamp in the log, if not found None
+
+        """
+        for line in log:
+            if re.search(snapname, line) and re.search(pattern, line):
+                return line.split(' ')[1]
+        return None
+
+    format = '%H:%M:%S.%f'
+    # Get the snapshoter-controller pod
+    pod_name = pod.get_csi_snapshotner_pod()
+    # get the logs from the csi-provisioner containers
+    logs = pod.get_pod_logs(
+        pod_name, namespace='openshift-cluster-storage-operator'
+    )
+    logs = logs.split("\n")
+
+    if status.lower() == 'start':
+        pattern = 'Creating content for snapshot'
+    elif status.lower() == 'end':
+        pattern = 'is bound to volumeSnapshotContent'
+
+    # Extract the time for the one PVC provisioning
```

Comment:
# Extract the time for the one PVC "snapshot". isn't it?

Reply:
fixed.